app/src/main/java/com/eslamwaheed/mvvmdemo/viewmodels/UnsplashViewModel.kt

```diff
@@ -16,11 +16,33 @@ class UnsplashViewModel @Inject constructor(private val repository: UnsplashRepo
 
     val unsplashSearchResponseLiveData = MutableLiveData<UnsplashSearchResponse>()
 
+    val photosLiveData = MutableLiveData<ArrayList<UnsplashSearchResponse.Result>>()
+
     fun searchPictures(query: String) {
         viewModelScope.launch(IO) {
             repository.getSearchResultStream(query).apply {
                 unsplashSearchResponseLiveData.postValue(this)
```

Comment:
you can use withcontext to switch to main thread but using postValue work fine

Reply:
I use withcontext when I need to talk a different thread ( like from background to main)